data/strings/strings.txt

```diff
@@ -2191,6 +2191,7 @@
     ru = Внутренний скрытый накопитель
     tr = Dahili özel depolama
     uk = Внутрішнє приватне сховище
+    hu = Belső privát tárolóhely
```

Comment:
It would be great to add it in alphabet order (en at the top is the exception) here and in other places.

Reply:
Next time.

Reply:
If you didn't learn yet how to amend commits and force-push, then you can create a second signed commit where necessary strings will be moved to the appropriate place. We'll squash them for you.

Learning git rebase and amend flow is highly recommended ;-)